string uri
string tf_prefix
float32 roll_trim
float32 pitch_trim
bool enable_logging
bool enable_parameters
LogBlock[] log_blocks
bool use_ros_time
bool enable_logging_imu
bool enable_logging_temperature
bool enable_logging_magnetic_field
bool enable_logging_pressure
bool enable_logging_battery
bool enable_logging_pose
bool enable_logging_packets
---
